common/shlibs

```diff
@@ -3325,6 +3325,8 @@ libvolume_key.so.1 volume_key-0.3.9_1
 librand48_r.so.0 rand48_r-0.1_1
 libxxhash.so.0 libxxHash-0.6.5_2
 libfwup.so.1 libfwup-11_1
+libfwupd.so.2 fwupd-1.3.9_1
```

Comment:
will anything in the repository ever link to these libraries? If not, they don't need to be added.

Reply:
Nothing links right now according to xrevshlib. I was adding it in simply because xbps-src warned that the shared object wasn't tracked in common/shlibs. I'll back that change out.

Reply:
That warning is kind of misleading. We try to keep the file small by only adding libraries other packages actually link to.

Reply:
Ahh, good to know. Thanks for clarifying!